cfme/tests/infrastructure/test_providers.py

```diff
@@ -492,3 +495,38 @@ def test_infra_discovery_screen(appliance):
                    "to_ip4": ips['to']})
         view.start.click()
         view.flash.assert_message(ips['msg'])
+
+
+@pytest.fixture
+def setup_provider_min_templates(request, appliance, provider, min_templates):
+    templates_yaml = len(provider.data.get('templates', {}))
+    if templates_yaml < min_templates:
+        pytest.skip(f'Number of templates defined in yaml for {provider} does not meet minimum '
+                    f'for test parameter {min_templates}, skipping and not setting up provider')
+    if len(provider.mgmt.list_host()) < min_templates:
+        pytest.skip(f'Number of templates on {provider} does not meet minimum '
+                    f'for test parameter {min_templates}, skipping and not setting up provider')
+    # Function-scoped fixture to set up a provider
+    setup_or_skip(request, provider)
+
+
+@pytest.mark.parametrize("min_templates", [2, 4])
+@pytest.mark.meta(blockers=[BZ(1784180, forced_streams=["5.10"])], automates=[1784180])
+def test_compare_provider_templates(appliance, setup_provider_min_templates, provider,
```

Comment:
Is this test case also providing coverage for BZ `1746449`?

Reply:
Yes it is. 1746449 was opened against 5.11.0. It was cloned to 5.11.z. as 1784180. I initially had the blocker and automates kwargs set to 1746449 with forced stream of 5.10, but this causes the tests to be skipped in 5.11.2 (also 5.10). If there is a better solution, I am all ears. maybe just add 1746449 to automates?

Reply:
Yup, just adding `1746449` to `automates` is all that's necessary here.

Reply:
Nevermind, since Mike added automatic clone fetching via `get_bz_info` this will be done automatically.

Reply:
As we discussed at your desk, It seems clones will be caught so I don't need to add 1746449 to automates. Thanks